src/updates-notes/change-log-data.json

```diff
@@ -1,7 +1,7 @@
 {
   "log": [
     {
-      "date": "09/06/23",
+      "date": "09/06/22",
```

Comment:
What's the reason for updating to the year 2022? Is our docs 1 year behind in regards to publishing?

Reply:
This is the date that the change was released, so it's today Sept 6, 2022.  Does not refer to the content being released.